README.md

```diff
@@ -105,3 +105,7 @@ Once you are set up and familiar with the example project, consider the followin
 6. Develop your scene and simulation scenario, placing Assets and configuring Components.
 
 Enjoy simulation with some of many [ROS2 packages](https://index.ros.org/packages/#humble) and projects in [ROS2 ecosystem](https://project-awesome.org/fkromer/awesome-ros2).
+
+## Acknowledgements
+
+This project was originally developed by [Robotec.ai](https://robotec.ai) in cooperation with [AWS Game Tech](https://aws.amazon.com/gametech/).
```

Comment:
Could you add https://aws.amazon.com/robomaker/ as well?

Reply:
Absolutely!